src/components/RewardCard.jsx

```diff
@@ -22,27 +22,27 @@ const Grid = styled.div`
 const Title = styled.h1`
   font-size: 24px;
   font-weight: 400;
-  color: ${p => p.theme.colors.lines};
+  color: ${p => p.theme.colors.text};
 `
 
 const Text = styled.h3`
   font-size: 18px;
   font-weight: 300;
   margin-top: -12px;
-  color: ${p => p.theme.colors.lines};
+  color: ${p => p.theme.colors.text};
 `
 
 const Description = styled.h3`
   font-size: 18px;
   font-weight: 300;
   text-align: center;
-  color: ${p => p.theme.colors.lines};
+  color: ${p => p.theme.colors.text};
 `
 
 const SubTitle = styled.p`
   font-size: 14px;
   font-weight: 700;
-  color: ${p => p.theme.colors.bar};
+  color: ${p => p.theme.colors.text};
```

Comment:
did you mention that this one should still be bar? or were you talking about a different one

Reply:
oh wait you're right LOL let me change it back